frontend/RaffleBay/RaffleBay/Controller/LoginViewController.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  LoginViewController.swift
+//  RaffleBay
+//
+//  Created by Meera Rachamallu on 2/13/20.
+//  Copyright © 2020 Meera Rachamallu. All rights reserved.
+//
+
+import Foundation
+import Alamofire_SwiftyJSON
+import Alamofire
+import SwiftyJSON
+func login_request(email: String, password: String) -> Int {
+    var response = "";
+    Alamofire.request("https://localhost:31337/api/users/login", method: .post)
```

Comment:
Should we configure Alamo to work in a production environment also or just keep everything on localhost?